RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -49,7 +49,7 @@ public override void Fix()
 
         private void RequestLocalCopyVariableName()
         {
-            using( var view = _factory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
+            using( var view = _dialogFactory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
```

Comment:
isn't this blowing up with a null reference exception in the tests when you pass a `null` factory to the inspection? You'll want to give it a `Mock<IAssignedByValParameterQuickFixDialogFactory>` object, something like this:

    var mock = new Mock<IAssignedByValParameterQuickFixDialogFactory>();
    mock.SetupGet(m => m.NewName).Returns(() => WhateverTheTestExpectsTheNewNameToBe);
    mock.SetupGet(m => m.IdentifierNamesAlreadyDeclared).Returns(() => WhateverTheTestNeedsToWorkWith);
    mock.SetupGet(m => m.IsCancelled).Returns(() => false); // or true, to test cancellation behavior

    var inspection = new AssignedByValParameterInspection(mockParser.State, mock.Object);